lib/statsd/instrument/backends/influxdb_backend.rb

```diff
@@ -0,0 +1,23 @@
+module StatsD::Instrument::Backends
+  # InfluxDB backend is actually a udp-compatible backend. However, it adds tag
+  # support. The tag is include in metric name with this format
+  # metricname#tag1=val1,tag2=val2:metricvalue|metrictype|@sample
+  # It's very similar to datadog backend, however, it move the tags into metric
+  # name. The backend where stat is flushed to is responsible to parse metric
+  # name for getting tag values
+  class InfluxDBBackend < StatsD::Instrument::Backends::UDPBackend
+
+    def generate_packet(metric)
+      command = "#{metric.name}"
+      if metric.tags
+        command << "##{metric.tags.join(',')}"
```

Comment:
Maybe this class should be responsible for escaping values if its a string.
https://docs.influxdata.com/influxdb/v0.9/write_protocols/write_syntax/#escaping-characters